controller pipelines integration test suite not compiling

**Describe the bug**
`controllers/pipelines/suite_integration_test.go` not compiling due to `argo/providers/stub/provider.go` moving to `argo/providers/stub/internal/provider.go` in this PR: https://github.com/sky-uk/kfp-operator/pull/391

**To Reproduce**
```sh
make integration-test-up
make integration-test
```

**Expected behavior**
tests build, run and pass

**Version and configuration**
`1cfeb3ef3913f9283cafc71e41952df9461e25dc`

**Logs**
```
# github.com/sky-uk/kfp-operator/controllers/pipelines
controllers/pipelines/suite_integration_test.go:19:2: no required module provides package github.com/sky-uk/kfp-operator/argo/providers/stub; to add it:
        go get github.com/sky-uk/kfp-operator/argo/providers/stub
FAIL    github.com/sky-uk/kfp-operator/controllers/pipelines [setup failed]
?
```

--- a/argo/providers/stub/cmd/main.go
+++ b/argo/providers/stub/cmd/main.go
@@ -2,7 +2,7 @@ package main
 
 import (
 	. "github.com/sky-uk/kfp-operator/argo/providers/base"
-	stub "github.com/sky-uk/kfp-operator/argo/providers/stub/internal"
+	"github.com/sky-uk/kfp-operator/argo/providers/stub"
 )
 
 func main() {

--- a/argo/providers/stub/provider.go
+++ b/argo/providers/stub/provider.go
@@ -1,4 +1,4 @@
-package internal
+package stub
 
 import (
 	"context"

